tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -587,8 +585,7 @@ async def test_nft_with_did_wallet_creation(two_wallet_nodes: Any, trusted: Any)
     spend_bundle = spend_bundle_list[0].spend_bundle
     await time_out_assert_not_none(5, full_node_api.full_node.mempool_manager.get_spendbundle, spend_bundle.name())
 
-    for _ in range(1, num_blocks):
-        await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
+    await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
```

Comment:
Unrelated to the change, do you really need the reward here?  If not then not receiving it makes the accounting for the expected balance that much simpler.